src/legacy/core_plugins/data/public/plugin.ts

```diff
@@ -119,7 +117,6 @@ export class DataPlugin implements Plugin<DataSetup, DataStart, {}, DataPluginSt
     return {
       ...this.setupApi!,
       indexPatterns: indexPatternsService,
-      search: this.search.start(savedObjects.client),
```

Comment:
Not a huge deal, but I'll probably be re-adding the search service to `core_plugins/data` in the next couple weeks while shimming SearchSource.